src/frontends/paddle/tests/CMakeLists.txt

```diff
@@ -47,9 +47,22 @@ ov_add_test_target(
 set(TEST_PADDLE_MODELS_DIRNAME ${TEST_MODEL_ZOO}/paddle_test_models)
 target_compile_definitions(${TARGET_NAME} PRIVATE -D TEST_PADDLE_MODELS_DIRNAME=\"${TEST_PADDLE_MODELS_DIRNAME}/\")
 
-set(PADDLEDET_OPS_URL "https://raw.githubusercontent.com/PaddlePaddle/PaddleDetection/release/2.1/ppdet/modeling/ops.py")
+execute_process(
+    COMMAND ${Python3_EXECUTABLE} -c "import paddle; print(paddle.__version__)"
+    OUTPUT_VARIABLE PADDLE_VERSION
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+message(STATUS "PaddlePaddle version: ${PADDLE_VERSION}")
+
+if(PADDLE_VERSION VERSION_GREATER_EQUAL "2.6.0" OR PADDLE_VERSION VERSION_EQUAL "0.0.0")
```

Comment:
Cmake will not automatically update the ops.py if ops.py is already downloaded and paddle version is changed. Recommend to save to different ${PADDLEDET_DIRNAME} for different paddle version.